src/QirRuntime/README.md

```diff
@@ -65,6 +68,11 @@ Some of the tests depend on Microsoft.Quantum.Simulator.Runtime library. To run
  from this repository or provide your own version of the library in a folder the OS would search during dynamic library
  lookup.
 
+Some of the tests use generated QIR (*.ll) files as build input. Currently the files are checked-in as part of the project
+ but in the future they will be replaced by automatic generation during build. To regenerate the files, run generateqir.py
+ or build/test scripts without specifying `noqirgen`. To use the checked-in files without regenerating them, run build/test
+ scripts with `noqirgen` argument.
+
```

Comment:
Minor:
The name `noqirgen` tells me that if `noqirgen` is specified, then no QIR generation (`.ll` files generation) happens. But if it is _not_ specified, then the QIR will be generated. To summarize, _the QIR is **generated by default**_.
This seems to me a perfect approach for the future, when the generation is implemented and stable, or if you know for sure when exactly the generation will be implemented.
For now, I assume, the generation is not complete (or is unstable). And if it is _not_ known for sure when the generation will be implemented, probably it makes sense to _not generate by default_. I.e. to rename the flag to `qirgen` (and if it is not specified then to _not_ generate).

Reply:
QIR generation is working on Windows and can/should be done by default. The only reason to provide this parameter is to allow suppression of QIR generation on Linux that seems to have issues at least for some people.